def.bzl

```diff
@@ -45,6 +45,23 @@ DEFAULT_LANGUAGES = [
     Label("//language/go:go_default_library"),
 ]
 
+def _valid_env_variable_name(name):
+    """ Returns if a string is in the regex [a-zA-Z][a-zA-Z0-9_]*
+
+    Given that bazel lacks support of regex, we need to implement
+    a poor man validation
+    """
+    for character in name:
+        if (character >= "a" and character <= "z") or \
+           (character >= "A" and character <= "Z") or \
+           (character >= "0" and character <= "9") or \
+           character == "_":
+            continue
+        return False
+    character = name[0]
```

Comment:
We should fail descriptively if `name` is empty, just check for `if not name` at the very top of this function.